scripts/privacy_scan_review.py

```
@@ -0,0 +1,384 @@
+        raise ReviewRecordError(f"owner-review record {label} is invalid")
+    return value
+
+
+def _canonical_path(value: object, *, label: str) -> str:
+    path = _string(value, label=label)
+    if (
+        path.startswith("/")
+        or "\\" in path
+        or path.startswith("redacted-path:")
+        or any(component in {"", ".", ".."} for component in path.split("/"))
+        or any(PATH_COMPONENT.fullmatch(component) is None for component in path.split("/"))
```

Ruff (0.16.2)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)